pil_analyzer/src/pil_analyzer.rs

```diff
@@ -651,6 +653,22 @@ namespace T(65536);
     col fixed p_reg_write_X_A = [0, 0, 0, 1, 0, 0, 0, 1, 0] + [0]*;
     col fixed p_reg_write_X_CNT = [1, 0, 0, 0, 0, 0, 0, 0, 0] + [0]*;
     { T.pc, T.reg_write_X_A, T.reg_write_X_CNT } in (1 - T.first_step) { T.line, T.p_reg_write_X_A, T.p_reg_write_X_CNT };
+"#;
+        let formatted = process_pil_file_contents::<GoldilocksField>(input).to_string();
+        if input != formatted {
+            for (i, f) in input.split('\n').zip(formatted.split('\n')) {
```

Comment:
I think we could replace that by pretty_assert

Reply:
For this I changed to a more appropriate test based on the input and expected formatted output (which just adds the namespaces)